frontend/src/elm/Models.elm

```diff
@@ -2,10 +2,12 @@ module Models exposing (..)
 
 import Routes
 import Login.Models as Login
+import Main.Models as Main
 
 
 type alias Model =
   { loginForm : Login.Model
+  , mainForm : Main.Model
```

Comment:
I think the name `mainForm` is not proper.
The `loginForm` really means the form of html. but the `mainForm` means the whole main page